setup.py

```diff
@@ -67,6 +67,8 @@
         'Operating System :: OS Independent',
     ],
     packages=find_packages(),
+    package_data={'tests': '*.xml'},
```

Comment:
Actually we intended to do the same as you later described "we probably want to either exclude the tests module". So the way to do it, is to modify the previous line to be `packages=find_packages(exclude=["tests"]),`. If you would kindly make that change and roll back this line, we would happily accept it!